.changeset/slow-flies-marry.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-events-backend-module-aws-sqs': patch
```

Comment:
Thinking it's best to split this out into a separate changeset, seeing as it's a much smaller change and unrelated to TechDocs

Reply:
Applied this myself so we get it shipped :grin: